phoenicis-javafx/src/main/java/com/playonlinux/javafx/views/common/Themes.java

```diff
@@ -19,24 +19,25 @@
 package com.playonlinux.javafx.views.common;
 
 public enum Themes {
-
-    DEFAULT("Default theme"),
-    DARK("Dark theme"),
-    BREEZE_DARK("Breeze Dark theme"),
-    HIDPI("HiDPI theme");
+    DEFAULT("Default theme", "default"),
+    DARK("Dark theme", "dark"),
+    BREEZE_DARK("Breeze Dark theme", "breeze"),
```

Comment:
Please name it breeze_dark. It's very well possible that someone implements the "normal" Breeze.